templates/DO-178C/README.md

```diff
@@ -1,3 +1,35 @@
 # StrictDoc template for DO-178C standard
 
-TBD: Instructions how to use.
+This DO-178C template provides an overview of the documents required as acceptable mean of compliance.
+
+The aim of the project is also to demonstrate the use of strictdoc integrating:
+- diagrams (asciiflow)
+- images
+- bitfields
+- doxygen
+- sphinx pdf and readthedoc publication
+
+For this purpose, a toy example is used.
+
+Requirements:
+```
+apt install ninja-build
+pip install sphinx
+pip install sphinx_rtd_theme
+pip install bit-field
+pip install cairosvg
+apt install doxygen
+pip install breathe
+apt install latexmk
+apt install texlive-latex-base
+apt install texlive-fonts-recommended
+apt install texlive-latex-extra
+```
+
+To rebuild spinx html and pdf:
```

Comment:
typo: `Sphinx`

please capitalize `HTML` and `PDF` to visually distinguish the formats better.